label_studio_ml/examples/tesseract/tesseract.py

```diff
@@ -44,15 +45,22 @@ def load_image(self, img_path_url, task_id):
             obj = S3_TARGET.Object(bucket_name, key).get()
             data = obj['Body'].read()
             image = Image.open(io.BytesIO(data))
+            image = ImageOps.exif_transpose(image)
             return image
         else:
+            cache_dir = os.path.join(self.MODEL_DIR, '.file-cache')
+            os.makedirs(cache_dir, exist_ok=True)
+            logger.debug(f'Using cache dir: {cache_dir}')
             filepath = self.get_local_path(
                 img_path_url,
+                cache_dir=cache_dir,
```

Comment:
Uses the same method as EasyOCR to avoid get_local_path cache conflicts.
https://github.com/HumanSignal/label-studio-ml-backend/blob/master/label_studio_ml/examples/easyocr/model.py#L100-L109